frontend/src/lib/components/AddToDashboard/AddToDashboardModal.tsx

```diff
@@ -61,7 +61,7 @@ const DashboardRelationRow = ({
                 className="overflow-hidden text-ellipsis whitespace-nowrap"
                 title={dashboard.name}
             >
-                {dashboard.name || 'Untitled'}
+                <span className={isAlreadyOnDashboard ? 'font-bold' : ''}>{dashboard.name || 'Untitled'}</span>
```

Comment:
`font-bold` is very bold, we usually go for semi-bold (600) max. But in here I'd say the red "Remove" vs. blue "Add" buttons are already a major indication, so I think the list would be cleaner without varying the font weight